File: Iniciante-Begginer/Python/#1101 - #1150/#1116-DividindoXporY.py
n = int(input())
divisao = 0

for i in range(1, n+1):
    x, y = map(int,input().split())
    if y == 0:
        print("divisao impossivel")
    else:
        divisao = x/y
        print("{:.1f}".format(divisao))

File: Iniciante-Begginer/C/#1101 - #1150/#1116-DividindoXporY.c
#include <stdio.h>


int main () {

  int n, x, y, i;
  float divisao = 0;
  scanf("%d", &n);


  for (i = 0; i < n; i++)
  {
    scanf("%d %d", &x, &y);
    if (y == 0)
    {

      printf("divisao impossivel\n");

    }
    else
    {

      divisao = (x*1.0)/(y);
      printf("%.1f\n", divisao);

    }
  }
}